modules/machineset-modifying.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// 
+// * machine_management/modifying-machineset.adoc
+
+[id="machineset-modifying_{context}"]
+= Modifying a MachineSet
+
+To make changes to a MachineSet, edit the MachineSet YAML. Then remove all machines associated with the MachineSet 
+by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number. 
+Changes you make to a MachineSet do not affect existing machines. 
+
+If you need to scale a MachineSet without making other changes, you do not need
+to delete the machines.
+
+[NOTE]
+====
+By default, the {product-title} router pods are deployed on workers. Because the router 
+is required to access some cluster resources, including the web console, 
+do not scale the worker MachineSet to `0` unless you first relocate the router pods.
```

Comment:
@jhou1 @ingvagabund What is the best way to relocate the router pods? Such as using node selectors?

Reply:
Out of my expertise, sorry :( Are the router pods deployed to every worker node or just some?